src/app/modules/users/upload-confirmation/upload-confirmation.component.html

```diff
@@ -0,0 +1,21 @@
+<div>
```

Comment:
Cant, we use the same confirmation modal??

Reply:
yes we can use , but for the purpose i kept it separate